lib/components/molecules/Popover/types/index.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import type { Node } from 'react';
+
+export type Props = {
+  popOverBody: Node | string,
+  popOverHeader: Node | string,
+  children: React$Element<*>,
+  isVisible: boolean,
+  wrapperBackground: string,
```

Comment:
wrapperBackground is useless, if someone wants to overwrite this component's style its not going to be background only. We should make way for enabling them to overwrite all CSS. Set class names in BEM terminology.

We are using this technique ${props => (props.inheritedStyles ? props.inheritedStyles : '')}; in all components. Please follow the same.

Reply:
Removed the unnecessary class and prop for css and aligned class naming with BEM.